charts/px-backup/templates/_helpers.tpl

```diff
@@ -36,9 +36,7 @@ Common labels
 */}}
 {{- define "px-backup.labels" -}}
 app.kubernetes.io/name: {{ template "px-backup.name" . }}
-helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
This is a required label according to helm, can we keep this? We can remove this from the template labels for the sts

https://helm.sh/docs/chart_best_practices/labels/

Reply:
Added